packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { WORKFLOW_MODEL_UID, STAGE_MODEL_UID } = require('../../constants/workflows');
+
+module.exports = ({ strapi }) => ({
+  async bootstrap() {
+    const wfCount = await strapi.entityService.count(WORKFLOW_MODEL_UID);
+    const stagesCount = await strapi.entityService.count(STAGE_MODEL_UID);
+
+    if (wfCount + stagesCount === 0) {
+      const defaultStages = require('../../constants/default-stages.json');
+      const defaultWorkflow = require('../../constants/default-workflow.json');
+
+      await strapi.query('admin::workflow-stage').createMany({ data: defaultStages });
+
+      const stages = await strapi
+        .query('admin::workflow-stage')
+        .findMany({ limit: 4, select: ['id'] });
```

Comment:
Is it a choice to make this static and not dynamic (talking about the 4 & the relations connect operations).
Can't we do smth like `limit: defaultStages.length` and connect: /* smth dynamic based on the id/idx */ ?